content/docs/csidriver/features/powerflex.md

```diff
@@ -522,6 +522,66 @@ Then run:
 this test deploys the pod with two ephemeral volumes, and write some data to them before deleting the pod.   
 When creating ephemeral volumes, it is important to specify the following within the volumeAttributes section: volumeName, size, storagepool, and if you want to use a non-default array, systemID.  
 
+## Consuming Existing Volumes with Static Provisioning
+
+You can use existent volumes from PowerFlex array as Persistent Volumes in your Kubernetes, perform these steps:
```

Comment:
minor. I would just rephrase as "To use existing volumes from PowerFlex array as Peristent volumes in your Kubernetes environment, perform these steps:"